inc/search.class.php

```diff
@@ -1140,7 +1140,7 @@ static function constructAdditionalSqlForMetacriteria($criteria = [],
          if (!in_array(getTableForItemType($m_itemtype), $already_link_tables)) {
             $FROM .= self::addMetaLeftJoin($data['itemtype'], $m_itemtype,
                                            $already_link_tables,
-                                           $criterion['value'] == "NULL" || strstr($criterion['link'], "NOT"),
+                                           true,
```

Comment:
The `addMetaLeftJoin` method seems to be used from `Search` only, I guess cwe can entirely remove the parameter that is now hardcoded, and clean the `addMetaLeftJoin` method a bit.

Reply:
+1 for removal